src/demo/demos.py

```diff
@@ -52,60 +51,65 @@ def bdc_demo():
 
 # evp demo
 def evp_demo():
+    data = get_database().load_preprocessed_data()
+
+    model_type_choices = [e for e in Predictors]
+    print("Which model type do you want to load")
+    for i, p in enumerate(Predictors):
+        print(f"({i}) : {p.value}")
+
+    choice = get_int_input("", range(0, len(model_type_choices)))
+    model_type = model_type_choices[choice]
+
     evp = EstimatedValuePredictor(
-        model_type=Predictors.LinearRegression,
-        model_path=input("Provide model path\n")
+        data=data,
+        model_type=model_type,
+        model_name=input("Provide model file name\n")
```

Comment:
When I do not pass any inputs and directly press Enter, we receive the following error: 
![Screenshot 2024-01-16 at 18 41 25](https://github.com/amosproj/amos2023ws06-sales-lead-qualifier/assets/45459787/ad7e6ca8-5d12-4e52-9ec1-56b17e71f4e3)

Reply:
Bug was reproducible and has been fixed